src/ui/main.cpp

```diff
@@ -94,7 +94,6 @@ int main(int argc, char *argv[])
     // There are no other instances: start a new server.
     a.startServer();
 
-    Editor::addEditorToBuffer();
```

Comment:
That was a mistake. The cose I submitted it worked too, but it should have read:

```
    // Apply font settings. Best to do that before any editors are opened to prevent
    // unneeded work. Otherwise we'd have to apply it to all editors currently created.
    QString fontFam = settings.value("overrideFontFamily", "").toString();
    int fontSz = settings.value("overrideFontSize", 0).toInt();

    Editor::setGlobalFontFamily(fontFam);
    Editor::setGlobalFontSize(fontSz);

    Editor::addEditorToBuffer();
```

This was meant to set the font override before any editors were added. But with your changes to how fonts are applied, this isn't necessary anymore.